src/gvmd.c

```diff
@@ -1833,6 +1834,12 @@ gvmd (int argc, char** argv)
           &feed_lock_path,
           "Sets the path to the feed lock file.",
           "<path>" },
+        { "feed-lock-timeout", '\0', 0, G_OPTION_ARG_INT,
+          &feed_lock_timeout,
+          "Sets the number of seconds to retry for if the feed is locked"
+          " in contexts (like migration or rebuilds) that do not retry"
+          " on their own (like automatic syncs). Defaults to 0.",
```

Comment:
I would explain the meaning of 0 too (i guess it is nowait).

Reply:
I've updated the text to mention that the default 0 means there is no retry.